drivers/docker/driver_test.go

```diff
@@ -1168,6 +1170,80 @@ func TestDockerDriver_CreateContainerConfig(t *testing.T) {
 	require.Equal(t, "org/repo:0.1", c.Config.Image)
 	require.EqualValues(t, opt, c.HostConfig.StorageOpt)
 }
+
+func TestDockerDriver_CreateContainerConfigWithRuntimes(t *testing.T) {
+	if !tu.IsTravis() {
+		t.Parallel()
+	}
+	if !testutil.DockerIsConnected(t) {
+		t.Skip("Docker not connected")
+	}
+	if runtime.GOOS != "linux" {
+		t.Skip("nvidia plugin supports only linux")
+	}
+	for _, testCase := range []struct {
```

Comment:
Define the cases outside the for-loop and use t.Run():
https://github.com/hashicorp/nomad/blob/6fb074e0e8a762d9d4fe2898233ad49bd59a0430/plugins/shared/structs/attribute_test.go#L12-L75